group-coordinator/src/main/java/org/apache/kafka/coordinator/group/modern/Assignment.java

```diff
@@ -69,20 +66,4 @@ public int hashCode() {
     public String toString() {
         return "Assignment(partitions=" + partitions + ')';
     }
-
-    /**
-     * Creates a {{@link Assignment}} from a {{@link ConsumerGroupTargetAssignmentMemberValue}}.
-     *
-     * @param record The record.
-     * @return A {{@link Assignment}}.
-     */
-    public static Assignment fromRecord(
```

Comment:
As I said, I think that we should keep it here.

Reply:
Done.